### PR TITLE
[Project Submission] Monaliza

### PR DESCRIPTION
**Pull request by georgeclarke23**
Adding Monaliza to Projects

### CODE REVIEW

**Review by autonome — CHANGES_REQUESTED** [review]
just need a screenshot and will be good to go!

File: projects/Monaliza.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+## Project Description
+
+The project is aimed to be a connector between instagram/social media and NFTs, enabling the average instagram/social media user to be able to mint their own NFTs and submit them straight to their instagram, utilising their fanbase and followers, as well as Opensea, and persist this to IPFS as a storage point. This would hopefully get the average person to be able to adopt blockchain without having to leave the comfort of their favourite platform and make their instagram posts/pictures/videos an NFT.
+
+## Project Status
+
+Under Development
+
+## Previews
+<!--Add some screenshots to give a preview of your product-->
```
[the remark this answers is not in the document]
Comment (MrFord123) [reply projects/Monaliza.md]: Added screenshot 👍

File: projects/Monaliza.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+### Team members  
+
+Darien Ford
+Jasmin Fiori
+George Clarke
+Aina Raherimanantsoa
+Seema Sharma
+
+## How the community can engage
+GitHub Discussion: <!--Start a disucssion with the community here: https://github.com/filecoin-project/community/discussions/new and attach the link!-->
```
Comment (autonome) [inline projects/Monaliza.md:80]: Apologies! i missed this earlier - please create the show-and-tell thread per the instructions here: https://github.com/ipfs/community#ecosystem-projects

**Review by autonome — CHANGES_REQUESTED** [review]

File: projects/Monaliza.md (hunk 2 above)
Comment (MrFord123) [reply projects/Monaliza.md:80]: No worries, I believe I've done this too. https://github.com/ipfs/community/discussions/493

File: projects/Monaliza.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+The project is aimed to be a connector between instagram/social media and NFTs, enabling the average instagram/social media user to be able to mint their own NFTs and submit them straight to their instagram, utilising their fanbase and followers, as well as Opensea, and persist this to IPFS as a storage point. This would hopefully get the average person to be able to adopt blockchain without having to leave the comfort of their favourite platform and make their instagram posts/pictures/videos an NFT.
+
+## Project Status
+
+Under Development
+
+## Previews
+
+![Landing Page Preview](/home/darien/Downloads/MonaSS.png)
```
Comment (autonome) [inline projects/Monaliza.md:19]: You need to upload to github - this path will not resolve.

File: projects/Monaliza.md (hunk 2 above)
Comment (autonome) [reply projects/Monaliza.md:80]: Yes, but please add the link here.

**Review by autonome — CHANGES_REQUESTED** [review]
A couple more changes required

File: projects/Monaliza.md (hunk 2 above)
Comment (MrFord123) [reply projects/Monaliza.md]: Hi, I've just added the link

File: projects/Monaliza.md (hunk 3 above)
Comment (MrFord123) [reply projects/Monaliza.md]: Hi, the path has now been edited too to resolve

**Comment by autonome** [comment]
looks great, thanks!

**Pull request merged**